### DISCUSSION TITLE
data: validationError.FAILED_VALIDATION

### DISCUSSION THREAD

**Discussion by scouech** [directus/directus#5237] [Archive]
Directus 60 

When I create a new collection and check Treat as single object, it seems that I can't edit the collection at all, I get the following error:

<img width="793" alt="Screenshot 2021-04-24 at 2 27 48 PM" src="https://user-images.githubusercontent.com/26603052/115958873-9e0ef080-a509-11eb-86ff-59d8798dbdb2.png">

Here are my fields and layout:
<img width="726" alt="Screenshot 2021-04-24 at 2 30 43 PM" src="https://user-images.githubusercontent.com/26603052/115958883-a7985880-a509-11eb-8eb6-b7c07b938209.png">

**Comment by rijkvanzanten** [comment]
This was resolved in https://github.com/directus/directus/pull/5209
